kernel/linux/config-5.2.17

```diff
@@ -4,7 +4,7 @@
 #
 
 #
-# Compiler: gcc (Gentoo 8.3.0-r1 p1.1) 8.3.0
+# Compiler: gcc (Gentoo 8.3.0-r2 p2) 8.3.0
```

Comment:
Why is this file changed ?

Reply:
I edited the configuration on my host computer. This file is auto-generated and this is the compiler's version, which has changed (on my host system) since the last update.

Afaik it's no big deal, it's useful for kernel developers to know the compiler's version when the configuration is copy pasted.

Reply:
Because evdev requires some kernel features to work properly.